kafka/init.lua

```diff
@@ -203,6 +205,31 @@ function Consumer:dump_conf()
     return self._consumer:dump_conf()
 end
 
+function Consumer:metadata(options)
+    if self._consumer == nil then
+        return
+    end
+
+    local timeout_ms = DEFAULT_TIMEOUT_MS
+    if options ~= nil and options.timeout_ms ~= nil then
+        timeout_ms = options.timeout_ms
+    end
+
+    return self._consumer:metadata(timeout_ms)
+end
+
+function Consumer:list_groups(group, timeout_ms)
```

Comment:
Maybe it's better to use options to configure timeout as in `Consumer:metadata` function in order to support one style

Reply:
Fixed as well